get_db_stats.py

```diff
@@ -0,0 +1,49 @@
+from os import getenv
+
+from pagai.services.database_explorer import POSTGRES, ORACLE, DatabaseExplorer
+from pagai.errors import OperationOutcome
+from sqlalchemy.exc import OperationalError
+import json
+
+def get_col_completion(owner, table_name, sorted, db_model):
```

Comment:
Where will this be called? Do you think we should make it a route (as for get_db_schema)?